src/app/component/partial/EventDetailsContainer/EventDetailsReRunAction.js

```diff
@@ -34,13 +35,19 @@ class EventDetailsReRunAction extends React.PureComponent {
   render() {
     const { children, event, client } = this.props;
 
+    // Unless this is a proxy entity target the specific entity
+    let subscriptions = [`entity:${event.entity.name}`];
```

Comment:
This would be nicer with a const declaration and a ternary.

Reply:
![image](https://user-images.githubusercontent.com/194892/55894703-768d9b00-5b6f-11e9-9d7d-c237dde37154.png)

---

In earnest though, I don't see a ton of value one way or the other. If you feel passionately about it and can find a linter rule to enforce it, I wouldn't be apposed to adding it.